docker/InitSubtract_deep_sort_and_compute.patch

```diff
@@ -0,0 +1,27 @@
+1c1
```

Comment:
Does this patch make sense to send upstream?

Reply:
It actually already is part of master. But since we work with kern, and kern uses the released version we need to patch as long as it's not in a new release.

Reply:
actually one option would be not to provide the patch, but pull it directly from github. Would that be more portable? In the end it's probably the same anyway but it may look cleaner.